docs/Wallet.md

```diff
@@ -99,10 +103,14 @@ Wallet re-scan solves two critical problems for BTCPay users:
 
 **Wallet import**: When users add a derivation scheme of a wallet that had transactions in the past (previously used wallet), BTCPay won't be able to show the balance and transactions from the past.
 
-![Wallet rescan progress](./img/BTCPayWallets5.png)
+![Wallet rescan progress](./img/wallet/WalletRescanProgress.png)
 
 Re-scan is a feature that solves both of these problems. Once the scan is complete, BTCPay Server will show the correct balance, along with the past transactions of the wallet.
 
 Wallet re-scan requires access to the full node which means that this function is only available for server owners.
 
 Users who use a third party host should use a newly generated xpub key and also use an external wallet like Electrum which allows them to increase the gap limit.
+
+### PSBT
+
+### Settings
```

Comment:
@Kukks need your help with this one, especially the accountkeypath and fingerprint if you can document this section when you get time, would help a lot.